pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -63,7 +64,16 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileNodeNetworkConfigurationPolicy{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	rollout, err := rollout.NewRollout(mgr.GetConfig(), mgr.GetScheme())
```

Comment:
I would prefer to remove rollout as a member of ReconcileNodeNetworkStateConfiguration since we are calling it at `Reconcile` only and in case of error it will re-reconcile, so we don't have to panic